source/tests/system/nirfmxlte_driver_api_tests.cpp

```diff
@@ -1,13 +1,14 @@
 #include <gtest/gtest.h>
 
 #include "device_server.h"
-#include "niRFmxLTE.h"
```

Comment:
I was going to explore this generally for all personalities but thought I'd go ahead and remove it in this PR for LTE. 

These headers shouldn't need to be included in the system level tests, right? The clients won't have access to them and we should just try to solely rely on the proto generated files for constants and things like that.

Reply:
Agreed. I just tried out removing it in my copy, where I'm adding the example-based tests, and I got it to work. I had to define float64, int32, and int64, and I had to replace some `_VAL_` constants. The latter is important, and is a good reason to stop including the C header.

Reply:
Agreed. I just tried out removing it in my copy, where I'm adding the example-based tests, and I got it to work. I had to define float64, int32, and int64, and I had to replace some `_VAL_` constants. The latter is important, and is a good reason to stop including the C header.

Reply:
Sounds good. I'll go through the other RFmx driver api tests and remove the headers in a separate PR.

Reply:
We also shouldn't need service headers, if you catch those. I think the only "legitimate" use of the driver headers is for expected error codes. But I haven't seen an instance of that in a while and it's not worth the cost that David mentions.